packages/shorebird_cli/lib/src/commands/login_command.dart

```diff
@@ -18,26 +18,35 @@ class LoginCommand extends ShorebirdCommand {
 
   @override
   Future<int> run() async {
-    if (auth.isAuthenticated) {
+    try {
+      await auth.login(prompt);
+    } on UserAlreadyLoggedInException catch (error) {
       logger
-        ..info('You are already logged in as <${auth.email}>.')
+        ..info('You are already logged in as <${error.email}>.')
         ..info("Run 'shorebird logout' to log out and try again.");
       return ExitCode.success.code;
+    } on UserNotFoundException catch (error) {
+      logger
+        ..err(
+          '''
+We could not find a Shorebird account for ${error.email}.''',
+        )
+        ..info(
+          """If you have not yet created an account, you can do so by running "${green.wrap('shorebird account create')}". If you believe this is an error, please reach out to us via Discord, we're happy to help!""",
```

Comment:
night but we've been using `lightCyan` to wrap commands in other places so might want to align on a single style for commands

Reply:
Fixed